src/Individual Article Page/ArticlePage.jsx

```diff
@@ -69,14 +73,27 @@ const ArticlePage = () => {
     setCommentStatus(false);
   };
 
+  const handleDeleteClick = (comment_id) => {
+    setDeleteCommentId(comment_id)
+    setDeleted(true)
+    setTimeout(() => setDeleted(false), 3000);
+    setComments((currentComments) => {
+      return currentComments.filter(
+        (comment) => comment.comment_id !== comment_id
+      );
+    });
+    deleteComments(comment_id)
+      .then(() => {})
+      .catch((err) => {
+        setComments((currentComments) => {
+          currentComments;
+          console.log(err, "error deleting comments");
```

Comment:
Likewise, we could have another state here which when set to true, displays a message saying there was an error deleting the comments, rather than just console logging it 🙂